app/controllers/DeleteAdditionalMetadataController.scala

```diff
@@ -15,10 +15,14 @@ import scala.concurrent.Future
 class DeleteAdditionalMetadataController @Inject() (
     val consignmentService: ConsignmentService,
     val customMetadataService: CustomMetadataService,
+    val displayPropertiesService: DisplayPropertiesService,
     val keycloakConfiguration: KeycloakConfiguration,
     val controllerComponents: SecurityComponents
 ) extends TokenSecurity {
 
+  private val closureDeletionMessage = "You are deleting closure metadata for the following files and setting them as open:"
+  private val descriptiveDeletionMessage = "You are deleting descriptive metadata for the following files:"
```

Comment:
Do you think we can move these messages to the property file?

Reply:
@vimleshtna do you mean the `/conf/message` file?

Reply:
Right.